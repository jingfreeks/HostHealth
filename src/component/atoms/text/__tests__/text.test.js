import React from 'react';
import {render} from '@testing-library/react-native';
import Text from '../text';

describe('Text Component', () => {
  it('Should work as expected to get snapshot', () => {
    const all = render(<Text />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
