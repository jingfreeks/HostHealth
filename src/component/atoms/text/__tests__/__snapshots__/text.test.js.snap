// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Component Should work as expected to get snapshot 1`] = `
<Text
  style={
    [
      {
        "color": "#0e3f6c",
        "fontFamily": "Avenir",
        "fontSize": 16,
        "fontWeight": "400",
      },
    ]
  }
/>
`;
